lib/Language/Slicer/Core.hs

```diff
@@ -293,6 +296,7 @@ data Trace = TExp (Syntax Trace)
                                             -- alternative in a case expression
            | TCaseR Trace (Maybe Var) Trace -- ^ Take "right constructor"
                                             -- alternative in a case expression
+           | TCaseExn Trace
```

Comment:
What does this stand for? Exception in the scrutinee? Can we write a program that triggers this?